docker-compose.local.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.3'
+services:
+    web:
+        build: .
+        image: geokrety:local
+        ports:
+            - "8000:80"
+        volumes:
+            - ${PWD}/website/:/var/www/html/:rw
+            - ${PWD}/configs/konfig-mysql.docker.php:/var/www/html/templates/konfig-mysql.php:ro
+            - ${PWD}/configs/konfig-local.tmpl.php:/var/www/html/templates/konfig-local.php:ro
+            - ${PWD}/configs/ssmtp.tmpl.conf:/etc/ssmtp/ssmtp.conf:ro
+        depends_on:
+            - db
+        environment:
+            MYSQL_HOST: db
+            MYSQL_USER: geokrety
+            MYSQL_PASSWORD: geokrety
+            MYSQL_DATABASE: geokrety-db
+
+    db:
+        image: mysql:5.7
+        volumes:
+            - ./db_data:/var/lib/mysql
+            # Initial DB schema structure
+            - ${PWD}/docker/mariadb:/tmp/database
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: ge0krety
```

Comment:
Password doesn't match the one from line 17

Reply:
because this is `root` password and good practice is to provide least required privilege. There is `geokrety` user with all the permissions, so no need for root.

Reply:
Ok, I missed that lines sorry :+1: